robotidy/transformers/MergeAndOrderSections.py

```diff
@@ -114,27 +127,34 @@ def from_last_section(node):
             *** Variables ****** Test Case ***
 
         """
-        if node.body:
-            last_statement = node.body[-1]
-            new_line = [Token(Token.EOL, "\n")]
-            if hasattr(last_statement, "body"):
-                if not last_statement.body:
-                    node.body[-1].body.append(EmptyLine.from_params(eol="\n"))
-                else:
-                    last_statement = last_statement.body[-1]
-                    if hasattr(last_statement, "end"):
-                        if last_statement.end:
-                            node.body[-1].body[-1].end = Statement.from_tokens(
-                                list(last_statement.end.tokens[:-1]) + new_line
-                            )
-                    else:
-                        node.body[-1].body[-1] = Statement.from_tokens(list(last_statement.tokens[:-1]) + new_line)
-            else:
-                node.body[-1] = Statement.from_tokens(list(last_statement.tokens[:-1]) + new_line)
+        # Empty sections, just *** Test Cases *** etc
+        if not node.body:
+            node.header.tokens = self.normalize_eol(node.header.tokens)
+            return node
+        # Settings, Variables or Comments
```

Comment:
Add blank line here or remove one in line 138, to remain consistent.

Reply:
Done